my_portfolio.html

```diff
@@ -1,13 +1,63 @@
-﻿<!DOCTYPE html>
-
+<!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>My Portfolio</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
     <link rel="stylesheet" href="style.css" />
 </head>
 <body>
-    <h1>Hello world</h1>
+    <div id="header-img"></div>
+    <div id="header-img-desktop"></div>
+    <div id="general-container">
+        <header id="nav-bar-ctn">
+            <div id="nav-bar">
+                <a>
+                    <p>Portfolio</p>
+                </a>
+                <a>
+                    <p>About</p>
+                </a>
+                <a>
+                    <p>Contact</p>
+                </a>
+            </div>
+        </header>
+        <nav id="toolbar">
+            <div id="menu-button">
+                <object data="icons/icon-menu.svg"></object>
+            </div>
+        </nav>
+        <section id="headline">
+            <h1 id="primary-text">
+                <p id="title-1">Hey there. I'm Gumaro.</p>
+                <p id="title-2">I'm a software developer</p>
+            </h1>
```

Comment:
Great job! There is just one suggestion for improvement:

Consider removing the `<p>` tag inside the `<h1>` tag. It is not semantically correct to have a `<p>` tag nested within an `<h1>` tag. By removing the `<p>` tag, you will ensure better adherence to semantic HTML standards. :+1: